### DISCUSSION TITLE
adding algerians tv

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#304] [Show and tell]
echorouk tv 
https://www.echoroukonline.com/live
ennahar tv
https://www.ennaharonline.com/live/

**Comment by Central302** [comment]
Echorouk
https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8
Ennahar is Not Work.

**Reply by ARAB-IPTV** [reply]
> Echorouk https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8 Ennahar is Not Work.

echorouk not work with me !

**Reply by BellezaEmporium** [reply]
> Echorouk https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8 Ennahar is Not Work.

Ennahar would be https://ennahar-live.dzsecurity.net/echo/EnnaharTV/playlist.m3u8

https://ennahar-live.dzsecurity.net/entv/EnnaharTV/playlist.m3u8

Funnily enough, this https://ennahar-live.dzsecurity.net/screen.jpg?0.6690813076150294 answers (if you add a referer to https://ennahar.dzsecurity.net/

**Reply by ARAB-IPTV** [reply]
> > Echorouk https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8 Ennahar is Not Work.
> 
> Ennahar would be https://ennahar-live.dzsecurity.net/echo/EnnaharTV/playlist.m3u8
> 
> https://ennahar-live.dzsecurity.net/entv/EnnaharTV/playlist.m3u8
> 
> Funnily enough, this https://ennahar-live.dzsecurity.net/screen.jpg?0.6690813076150294 answers (if you add a referer to https://ennahar.dzsecurity.net/

this url work with me on tivimate and kodi 

#EXTINF:-1  tvg-logo="https://i.ibb.co/x7JWhH5/ennahar.png" ,Ennahar News النهار 
https://ennahar-live.dzsecurity.net/fullres/EnnaharTV_First/chunks.m3u8|Referer=https://echorouk.dzsecurity.net/embed/echorouk.php

**Reply by ARAB-IPTV** [reply]
> > Echorouk https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8 Ennahar is Not Work.
> 
> Ennahar would be https://ennahar-live.dzsecurity.net/echo/EnnaharTV/playlist.m3u8
> 
> https://ennahar-live.dzsecurity.net/entv/EnnaharTV/playlist.m3u8
> 
> Funnily enough, this https://ennahar-live.dzsecurity.net/screen.jpg?0.6690813076150294 answers (if you add a referer to https://ennahar.dzsecurity.net/

ismael , can you get the link of samira tv from this site 
http://www.babalweb.net/television/SAMIRATV-samira-tv-diffusion-en-direct-live-streaming.html

**Reply by BellezaEmporium** [reply]
> > > Echorouk https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8 Ennahar is Not Work.
> > 
> > 
> > Ennahar would be https://ennahar-live.dzsecurity.net/echo/EnnaharTV/playlist.m3u8
> > https://ennahar-live.dzsecurity.net/entv/EnnaharTV/playlist.m3u8
> > Funnily enough, this https://ennahar-live.dzsecurity.net/screen.jpg?0.6690813076150294 answers (if you add a referer to https://ennahar.dzsecurity.net/
> 
> ismael , can you get the link of samira tv from this site http://www.babalweb.net/television/SAMIRATV-samira-tv-diffusion-en-direct-live-streaming.html

http://192.34.153.36:7777/SamiraTV_HD.m3u8?user=sgls540839&session=966647e26f0cd657ff9fb233cc0ce16b92128728871b94211684ce07cc3b771e0412b8d55803f734&v=4

**Reply by BellezaEmporium** [reply]
https://www.glwiz.com/Pages/Player/Ajax.aspx?action=getStreamURL&ClusterName=zixi-glwiz-mobile&RecType=4&itemName=Manoto&ScreenMode=0

basically ripping links from this website : https://www.glwiz.com/

**Comment by Sphinxroot** [comment]
#EXTVLCOPT:http-referrer=https://echorouk.dzsecurity.net/embed/echorouk.php

**Reply by ARAB-IPTV** [reply]
> #EXTVLCOPT:http-referrer=https://echorouk.dzsecurity.net/embed/echorouk.php

i do like this , but dont work 

#EXTINF:-1 group-title="" tvg-id="" tvg-logo="",echorouk
#EXTVLCOPT:http-referrer=https://echorouk.dzsecurity.net/embed/echorouk.php
#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36
https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8

**Reply by Sphinxroot** [reply]
#EXTINF:-1 ,Echorouk TV 360p in kodi
https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8|Referer=https://echorouk.dzsecurity.net/embed/echorouk.php
![ECHO](https://user-images.githubusercontent.com/47762050/151201816-55421083-4fc3-4dad-a7c7-f69d283a3e69.PNG)

**Reply by ARAB-IPTV** [reply]
ah okay , i dont paly it with kodi , i just palyed it with tivimate and vlc

**Reply by ARAB-IPTV** [reply]
can you give me the url stream of this ; echorouk news 
https://www.echoroukonline.com/live-news

**Reply by Sphinxroot** [reply]
#EXTINF:-1 ,Echorouk News 360p
https://echorouk-live-tv.dzsecurity.net/echo/EchoroukNews/playlist.m3u8|Referer=https://echorouk.dzsecurity.net/embed/news.php
![newse](https://user-images.githubusercontent.com/47762050/151296675-44d762cf-2b7a-4878-a861-51ed8e3db26b.PNG)

**Reply by Sphinxroot** [reply]
> ah okay , i dont paly it with kodi , i just palyed it with tivimate and vlc

unsure if tivimate handle referrer tho ..
 vlc ,  kodi (Playlistloader) does

**Reply by ARAB-IPTV** [reply]
> > ah okay , i dont paly it with kodi , i just palyed it with tivimate and vlc
> 
> unsure if tivimate handle referrer tho .. vlc , kodi (Playlistloader) does

its work good on tivimate, thank you so much

**Comment by Sphinxroot** [comment]
ennahar tv
https://www.ennaharonline.com/live/

https://ennahar-live.dzsecurity.net/entv/EnnaharTV/playlist.m3u8
#EXTVLCOPT:http-referrer=https://echorouk.dzsecurity.net/embed/ennahar.php

no mediamanifest found  Offlline or not 24/7

**Comment by ARAB-IPTV** [comment]
hello 
can you add samira tv from shahid.net 
https://shahid.mbc.net/ar/livestream/TV-%D8%B3%D9%85%D9%8A%D8%B1%D8%A9/livechannel-960055

**Reply by BellezaEmporium** [reply]
https://d2p6676ajpsg1j.cloudfront.net/out/v1/2daff8b433344d659bd5079224afc3ab/index.m3u8

**Reply by BellezaEmporium** [reply]
For World Cup MBC : https://shls-live-ak.akamaized.net/out/v1/0965e4d7deae49179172426cbfb3bc5e/index.m3u8

**Reply by BellezaEmporium** [reply]
Gulli Bil Arabi : https://shls-gulli-bil-arabi-prod-dub.shahid.net/out/v1/5454d215afba410c90b233f400730958/index.m3u8

**Comment by ARAB-IPTV** [comment]
adding el watania tv algerie 
https://elwataniatv.dz/%d9%85%d8%a8%d8%a7%d8%b4%d8%b1/
when i search for the m3u8 link of the channel i get this links
https://stream.castr.com/62c18c3f14d09a0b7e5355a5/live_95f873a0495c11edafed1198d684e90d/index.m3u8?check=1
https://stream.castr.com/62c18c3f14d09a0b7e5355a5/live_95f873a0495c11edafed1198d684e90d/tracks-v1a1/mono.m3u8
https://stream.castr.com/62c18c3f14d09a0b7e5355a5/live_95f873a0495c11edafed1198d684e90d/tracks-v1a1/2022/12/03/08/00/40-08333.ts
but dont work !!

**Reply by BellezaEmporium** [reply]
Because, like all Castr links, it needs a referer.

**Reply by ARAB-IPTV** [reply]
how can i do a reffer for the channel !
